src/components/scenes/LoginScene.tsx

```diff
@@ -162,11 +162,15 @@ class LoginSceneComponent extends React.PureComponent<Props, State> {
 
     const { notificationPermissionsInfo } = this.state
     if (notificationPermissionsInfo) {
-      const newSettings = await this.props.dispatch(setDeviceSettings(notificationPermissionsInfo.notificationOptIns))
-      this.props.dispatch({
-        type: 'NOTIFICATION_SETTINGS_UPDATE',
-        data: serverSettingsToNotificationSettings(newSettings)
-      })
+      try {
+        const newSettings = await this.props.dispatch(setDeviceSettings(notificationPermissionsInfo.notificationOptIns))
+        this.props.dispatch({
+          type: 'NOTIFICATION_SETTINGS_UPDATE',
+          data: serverSettingsToNotificationSettings(newSettings)
+        })
+      } catch (e) {
+        console.error(String(e))
```

Comment:
No need to cast here because console.error should take any. Unlesss console.error was mutated as a global and the API is unexpected in the GUI?

Reply:
The String cast was recommended by @swansontec in our huddle last week. I'll leave as is for now and discuss further. The main goal is to prevent the red error dropdown. @peachbits I didn't repo this either, but based on the text in the red error, the cleaner `asDevicePayload` called in `setDeviceSettings` is the only cleaner that could generate this error and it's always handled except in the call to `setDeviceSettings` from the code changed above.